app/main.py

```diff
@@ -1,3 +1,12 @@
 def cache(func):
-    # Write your code here
-    pass
+    arr = []
+
+    def inner(*args, **kwargs):
+        if args in arr:
+            print("Getting from cache")
```

Comment:
Function should return result in any case. But it just prints `Getting from cache` when `args` are in `arr1`